subsys/cpp/Kconfig

```
@@ -37,10 +37,15 @@ config STD_CPP17
 	help
 	  2017 C++ standard, previously known as C++0x.
 
-config STD_CPP2A
-	bool "C++ 2a"
+config STD_CPP20
+	bool "C++ 20"
 	help
-	  Next revision of the C++ standard, which is expected to be published in 2020.
+	  2020 C++ standard, previously known as C++2A
```

Nit: Just to be consistent with the rest of these.
```suggestion
	  2020 C++ standard, previously known as C++2A.
```